com.unity.shadergraph/Editor/Importers/ShaderGraphImporter.cs

```diff
@@ -26,7 +26,7 @@ namespace UnityEditor.ShaderGraph
     // This ifdef can be removed once V2 is the only option.
     [ScriptedImporter(115, Extension, -902)]
 #else
-    [ScriptedImporter(48, Extension, -902)]
+    [ScriptedImporter(49, Extension, -902)]
```

Comment:
Why only update one of these numbers? You need to update both if the serialization actually requires a reimport on upgrade.

Reply:
correct, both should be updated. My mistake :)

Reply:
Thanks Liz! Will amend this 👍

Reply:
Ah, no, you changed the entirely wrong thing. you need:

```
 [ScriptedImporter(116, Extension, -902)]
#else
    [ScriptedImporter(49, Extension, -902)]
```

The first number is the call is the version, unique to SG, which must be bumped by 1 independently for each branch of the #if. The second number is the global importer order which should be the same for both branches of the #if, and should not change.

Reply:
Ahh that makes sense 👍 
Currently in master the state of the Importer parameters is as such:

```
    [ScriptedImporter(121, Extension, -902)]
#else
    [ScriptedImporter(50, Extension, -902)]
#endif
```

post the merge into this branch and bumping the appropriate version numbers: 

```
    [ScriptedImporter(122, Extension, -902)]
#else
    [ScriptedImporter(51, Extension, -902)]
```